src/atopile/address.py

```diff
@@ -17,6 +17,20 @@ class AddrStr(str):
     Represents address strings
     """
 
+    def _split(self) -> tuple[Path, str]:
+        path, module = self.rsplit(":", 1)
+        return Path(path), module
+
+    @property
+    def file_path(self) -> Path:
+        path, _ = self._split()
+        return path
+
+    @property
+    def module_path(self) -> str:
```

Comment:
This is incomplete

Addresses can also have a double-semicolon to represent an instance path in addition to a module path